lib/rxjs/rxjs.ts

```diff
@@ -0,0 +1,140 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+Zone.__load_patch('rxjs', (global: any, Zone: ZoneType, api: _ZonePrivate) => {
+  let rxjs;
+
+  const subscribeSource = 'rxjs.subscribe';
+  const nextSource = 'rxjs.Subscriber.next';
+  const errorSource = 'rxjs.Subscriber.error';
+  const completeSource = 'rxjs.Subscriber.complete';
+  const unsubscribeSource = 'rxjs.Subscriber.unsubscribe';
+
+  try {
+    rxjs = require('rxjs');
```

Comment:
`require` statement will not work in browser.

So far the `.js` files have been stand alone code which can execute in global context, this means that we can't rely on ES6 `import` to get a hold of `Rx`. 

There is also the matter that we have to run the code at a specific time. We have to load `Zone.js` first than `Rx` and only than we can do `Rx` patching but before the other code gets it. 

I think the solution here is to package this file as `umd` that way it can be used in both cases.
It would mean that at some point one would have to import it as: `zone.js/rxjs-patch`